sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/TestUtils.java

```diff
@@ -96,7 +146,11 @@ public static String getConnectionString(boolean withSas) {
      * @return The fully qualified domain name for the service bus resource.
      */
     public static String getFullyQualifiedDomainName() {
-        return System.getenv("AZURE_SERVICEBUS_FULLY_QUALIFIED_DOMAIN_NAME");
+        return getPropertyValue("AZURE_SERVICEBUS_FULLY_QUALIFIED_DOMAIN_NAME");
+    }
+
+    public static String getEndPoint() {
+        return getPropertyValue("AZURE_SERVICEBUS_EDNPOINT_SUFFIX", ".servicebus.windows.net");
```

Comment:
Can you please change the value to `servicebus.windows.net`?
This way you don't have to do getEndpoint().substring(1) everywhere else, and anyplace where you need the "." it can be added